docs/pages/access-controls/access-request-plugins/ssh-approval-opsgenie.mdx

```diff
@@ -0,0 +1,114 @@
+---
+title: Access Requests with Opsgenie
+description: How to set up Teleport's Opsgenie plugin for privilege elevation approvals.
+---
+
+With Teleport's Opsgenie integration, engineers can access the infrastructure
+they need to resolve alerts quickly—without longstanding admin permissions
+that can become a vector for attacks.
+
+Teleport's Opsgenie integration allows you to treat Teleport Role Access
+Requests as Opsgenie alerts, notify the appropriate on-call team, and
+approve or deny the requests via Teleport. You can also configure the plugin to
+approve Role Access Requests automatically if the user making the request is on
+the on-call team for a service affected by an alert.
+
+This guide will explain how to set up Teleport's Access Request plugin for
+Opsgenie.
+
+(!docs/pages/includes/plugins/diagram.mdx api="Opsgenie" action="Listen for Access Requests\nModify Access Requests" messages="Send notifications\nGet alert status" !)
+
+## Prerequisites
+
+(!docs/pages/includes/commercial-prereqs-tabs.mdx!)
+
+- An Opsgenie account with the ability to create API keys with the 'read' and,
+  'create and update' access rights.
+
+  You can create API keys by going to the Opsgenie settings page. Navigating to
+  the 'APP settings' section and selecting API key management.
+
+- This feature is currently only available for Cloud isntances.
+
+- (!docs/pages/includes/tctl.mdx!)
+
+## Step 1/5. Create services
+
+To demonstrate the Opsgenie plugin, create an Opsgenie team named
+`teleport-access-request-notifications`.
+
+We will configure the Opsgenie plugin to create an alert for the
+`teleport-access-request-notifications` team when certain users
+create an Access Request.
+
+## Step 2/5. Define RBAC resources
+
+The Teleport Opsgenie plugin works by receiving Access Request events from the
+Teleport Auth Service and, based on these events, interacting with the Opsgenie
+API.
+
+### Create a user who will request access
+
+Create a user called `myuser` who has the `editor-requester` role. Later in this
+guide, you will create an Access Request as this user to test the Opsgenie
+plugin:
+
+```code
+$ tctl users add myuser --roles=editor-requester
```

Comment:
```suggestion
$ tctl users add myuser --roles=requester
```

Reply:
Also, since we're explaining how to setup hosted integration which basically is done via browser, I wonder if we should show how to create a user in the web UI instead of asking users to use tctl commands.

@ptgott What do you think?

Reply:
I haven't looked at this guide yet, but if there are no other `tctl` commands, I think it would be great to show how to create a user via the browser so the reader doesn't have to switch contexts. Otherwise, I think we can leave the instructions as-is since the reader will already have a terminal open and be authenticated to Teleport.